scipy/stats/_entropy.py

```diff
@@ -46,20 +47,20 @@ def entropy(pk: np.typing.ArrayLike,
 
     Returns
     -------
-    S : {float, array_like}
+    H : {float, array_like}
```

Comment:
No need for this to match what's inside the function. However, it would probably be better if it were neither the symbol for entropy or relative entropy; so maybe revert this change and rename the set below?
```suggestion
    S : {float, array_like}
```